app/src/main/res/layout/activity_appearance.xml

```diff
@@ -102,6 +102,170 @@
                 app:primaryTextTruncated="false"
                 app:secondaryText="@string/settingsAddressBarPositionTop" />
 
+            <com.duckduckgo.common.ui.view.listitem.SectionHeaderListItem
+                android:id="@+id/omnibarTypeSettingsTitle"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:primaryText="@string/settingsAddressBarPositionHeaderTitle" />
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:id="@+id/omnibarTypeToggleContainer"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/keyline_1"
+                android:layout_marginBottom="@dimen/keyline_2"
+                android:layout_marginHorizontal="@dimen/keyline_4">
+
+                <LinearLayout
+                    android:id="@+id/topOmnibarContainer"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:gravity="center"
+                    android:orientation="vertical"
+                    android:padding="@dimen/keyline_2"
+                    android:background="@drawable/selectable_rounded_ripple"
```

Comment:
When we added a similar toggle layout to `AI Features` settings menu, we dropped the click ripple - I think we should do the same here. Could you double-check with design if needed?